### DISCUSSION TITLE
Can't install the mrgsolve package on Windows

### DISCUSSION THREAD

**Discussion by cjli65** [metrumresearchgroup/mrgsolve#1016] [Q&A]
I tried this command: `remotes::install_github("metrumresearchgroup/mrgsolve@dev")` but got this error 
```
Downloading GitHub repo metrumresearchgroup/mrgsolve@dev
Error in utils::download.file(url, path, method = method, quiet = quiet,  : 
  cannot open URL 'https://api.github.com/repos/metrumresearchgroup/mrgsolve/tarball/dev'
```
can anyone help me?

**Comment by kylebaron** [comment]
Hi @cjli65  - 

The url on README for installing development version via remotes was wrong; we are keeping development version in `develop` branch now. I've fixed the README. 

Could you try this instead:

```r
remotes::install_github("metrumresearchgroup/mrgsolve@develop")
```

Ref: https://github.com/metrumresearchgroup/mrgsolve#installation

Thanks, 
Kyle

**Comment by cjli65** [comment]
Thank you so much Kyle! The new command worked, and mrgsolve is now installed.Thanks again!

Austin
